airflow/models/taskinstance.py

```diff
@@ -950,7 +950,10 @@ def signal_handler(signum, frame):
                 Stats.incr('ti_successes')
             self.refresh_from_db(lock_for_update=True)
             self.state = State.SUCCESS
-        except AirflowSkipException:
+        except AirflowSkipException as e:
+            # log only if exception has any arguments to prevent log flooding
+            if e.args:
+                self.log.warning(e)
```

Comment:
Maybe this should be at info level? It doesn't really seem like a "warning" to me.

WDYT?

Reply:
Agreed. Since task skipping is not so much of a warning in airflow. I have made the change. Thanks for the tip.